src/Main.java

```diff
@@ -1,6 +1,54 @@
 public class Main {
     public static void main(String[] args) {
         System.out.println("Задача №1");
-
+        for (int i = 1; i <= 10; i++) {
+            System.out.println(i);
+        }
+        System.out.println("Задача №2");
+        for (int i = 10; i >= 1; i--) {
+            System.out.println(i);
+        }
+        System.out.println("Задача №3");
+        for (int i = 0; i <= 17; i = i + 2) {
+            System.out.println(i);
+        }
+        System.out.println("Задача №4");
+        for (int i = -10; i <= 10; i++) {
+            System.out.println(i);
+        }
+        System.out.println("Задача №5");
+        for (int i = 1904; i <= 2096; i = i + 4) {
+            System.out.println(i + " год является високосным");
+        }
+        System.out.println("Задача №6");
+        for (int i = 7; i <= 98; i = i + 7) {
+            System.out.println(i);
+        }
+        System.out.println("Задача №7");
+        for (int i = 1; i <= 512; i = i * 2) {
+            System.out.println(i);
+        }
+        System.out.println("Задача №8");
+        int cash = 29000;
+        int sum = 0;
+        for (int i = 0; i <= 12; i++) {
+            sum = sum + cash;
+            System.out.println("Месяц " + i + " сумма накоплений равна " + sum);
+        }
+        System.out.println("Задача №9");
+        int cash1 = 29000;
+        int sum1 = 0;
```

Comment:
зачем заводить новые переменные? почему бы просто не обнулить и не переиспользовать уже объявленные выше?
```suggestion
        sum = 0;
```